src/ui/edit/missing-framework-elements-dialog.c

```diff
@@ -299,10 +302,13 @@ bt_missing_framework_elements_dialog_apply (const
       GST_INFO ("enlarging to %d bytes", length);
       machine_ignore_list = g_realloc (machine_ignore_list, length);
     }
+	gint length_total = length;
     if (machine_ignore_list) {
       ptr = &machine_ignore_list[offset];
       for (node = edit_elements; node; node = g_list_next (node)) {
         length = g_sprintf (ptr, "%s,", (gchar *) (node->data));
+		length_total -= length;
+		g_assert(length_total >= 0);
```

Comment:
the g_assert will fail the program without much context for the user. The code before calculates the length. It would probably a good idea to extract the code into a helper and add a test, which I can take a look at. Please revert this part for now.

Reply:
Will do